### PR TITLE
get queue names from the contract addresses if we can

### LINKED ISSUE #198: have a queue object for each scheme that we know, and add the name
We can "fake" the information that is hard to get in the subgraph. This addresses issues https://github.com/daostack/subgraph/issues/195 and https://github.com/daostack/subgraph/issues/202

### PR DESCRIPTION
**Pull request by jellegerbrandy**
shoud fix #198

### CODE REVIEW

File: src/queue.ts [hunk 1]
```diff
@@ -25,26 +26,31 @@ export class Queue {
...
-         name
-       }
+       name
+       address
      }
    }`
-    const itemMap = (item: any): Queue => {
+    const itemMap = (item: any): Queue|null => {
+      const name = item.name || context.getContractName(item.address)
+      console.log(name === options.name)
```
Comment (tibetsprague) [inline src/queue.ts]: remove consoles

File: src/queue.ts [hunk 2]
```diff
@@ -75,7 +81,7 @@ export class Queue {
             address
             name
             dao { id }
-            canDelegateCall
+            canDeltegateCall
```
Comment (tibetsprague) [inline src/queue.ts]: this is mispelled, on purpose?

File: src/queue.ts [hunk 3]
```diff
@@ -89,30 +95,23 @@ export class Queue {
 
     const itemMap = (item: any): IQueueState|null => {
       if (item === null) {
-        // no gpqueue was found - we return null
-        return null
+        // no gpqueue was found - we construct one with basic default values
```
Comment (tibetsprague) [inline src/queue.ts]: not a gpqueue anymore

File: src/queue.ts (hunk 2 above)
Comment (jellegerbrandy) [reply src/queue.ts]: not on purpose... not sure why it does not error, though...

**Pull request merged**